docs/docs/faq.md

```diff
@@ -152,6 +152,9 @@ Intel will whitelist your signing key and the release mode then becomes availabl
     object to the clients. The Conclave client libraries embed the necessary certificates to verify Intel's signature over
     this data, and the integrity of the object is checked automatically when it's deserialized.
 
+!!! Important
```

Comment:
The documentation shouldn't be introducing the concept of EPID anymore. Instead where applicable it should be changed to DCAP

Reply:
Ok, so if we are deprecating we don't even want to give instructions for EPID, then? I.e. we want to "pretend" that it is not existing any more in the instructions, right? Why not demising then?

Reply:
Yes, because otherwise documentation implies customer support if they have issues, which is not something we want to be doing now.